### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by ali-gillani**

### CODE REVIEW

File: task1.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def search_word(x, my_list):
+    for y in my_list:
```
Comment (UsmanAbbasi1) [inline task1.py]: try to think of pythonic solution for this task.

Comment (ali-gillani) [reply task1.py]: Sir I had changed the code.Now have a look at it.

File: task1.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def search_word(element, my_list):
+    if element in my_list:
+        return True
+    else:
+        return False
+
+
+print("Enter The Value")
+value = input()
+my_list1 = ['Lahore', 'Pakistan', 'Karachi', 'Peshawar', 'Quetta']
```
Comment (UsmanAbbasi1) [inline task1.py]: Read the words from file and and then add them in list.

Comment (ali-gillani) [reply task1.py]: ok

**Pull request merged**